packages/falter-common/files-common/etc/config/freifunk

```diff
@@ -104,6 +104,12 @@ config 'defaults' 'wifi_iface'
 	option 'bssid' '12:CA:FF:EE:BA:BE'
 	option 'mcast_rate' '6000'
 
+config 'defaults' 'wifi_iface_5'
+	option 'mode' 'adhoc'
+	option 'encryption' 'none'
+	option 'bssid' '12:CA:FF:EE:BA:BF'
```

Comment:
I think that is a typo, isn't it? Shouldn't it be `12:CA:FF:EE:BA:BE` ?

Reply:
Nope. 5 GHz and 2.4 ghz have different bssids.
But I'm still not sure why setting bssid at all. ;)

Reply:
The bssid for adhoc in 5GHz on channel 36 should be ```02:36:CA:FF:EE:EE``` and other bssids are listed at https://wiki.freifunk.net/Freifunk_Berlin_Pberg:Kanal_BSSID

Reply:
Just for a bit of background as to why each channel has it's own essid and bssid...

It would obviously be nice if we could just mesh on any channel with a single standard essid and bssid, but there is a problem with adhoc which has to deal with overlapping channels.  How I understand it, if there are two adhoc nodes on, for example, channel 1 and 2, and both have the same essid and bssid, then they will make an adhoc connection to each other.  This is problematic because communication between channel 1 and 2 are __not__ good.

In order to remediate this problem, some smart freifunkers decided to have separate essid's and bssid's for each channel to guarantee that the problem described above never happens.

Reply:
> The bssid for adhoc in 5GHz on channel 36 should be `02:36:CA:FF:EE:EE` and other bssids are listed at https://wiki.freifunk.net/Freifunk_Berlin_Pberg:Kanal_BSSID

here too https://wiki.freifunk.net/Archiv:Kamikaze/Meshwizard/BSSID-Schema